docs/docs/cmd/spo/roledefinition/roledefinition-add.md

```diff
@@ -0,0 +1,43 @@
+# spo roledefinition add
+
+Adds a new roledefinition to web
+
+## Usage
+
+```sh
+m365 spo roledefinition add [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: URL of the site to which role should be added
+
+`-n, --name <name>`
+: role definition name
+
+`-d, --description [description]`
+: role definition description
+
+`--rights [rights]`
+: A case sensitive string array that contain the permissions needed for the custom action. Allowed values `EmptyMask,ViewListItems,AddListItems,EditListItems,DeleteListItems,ApproveItems,OpenItems,ViewVersions,DeleteVersions,CancelCheckout,ManagePersonalViews,ManageLists,ViewFormPages,AnonymousSearchAccessList,Open,ViewPages,AddAndCustomizePages,ApplyThemeAndBorder,ApplyStyleSheets,ViewUsageData,CreateSSCSite,ManageSubwebs,CreateGroups,ManagePermissions,BrowseDirectories,BrowseUserInfo,AddDelPrivateWebParts,UpdatePersonalWebParts,ManageWeb,AnonymousSearchAccessWebLists,UseClientIntegration,UseRemoteAPIs,ManageAlerts,CreateAlerts,EditMyUserInfo,EnumeratePermissions,FullMask`. Default `EmptyMask`
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
+
+The `--rights` option accepts **case sensitive** values.
```

Comment:
Let's change the wording of `case sensitive` to `case-sensitive`

Reply:
done